app/pages/docs/tradeoffs.mdx

```diff
@@ -1,58 +1,56 @@
 ---
-title: Tradeoffs
-sidebar_label: Tradeoffs
+title: Afwegingen
+sidebar_label: Afwegingen
 ---
 
-As you know, everything has tradeoffs. So what are the tradeoffs of Blitz?
+Zoals je weet heeft alles afwegingen, maar wat zijn nu juist de afwegingen van Blitz?
 
-## Framework Lock-in {#framework-lock-in}
+## Framework lock-in {#framework-lock-in}
 
-Every framework has a certain level of lock-in. If you build your app with React, it's very locked into React.
+Elk framework heeft een bepaalde mate van lock-in. Als je jouw app met React bouwt, is deze nauw verweven met React.
 
-Blitz's level of lock-in is one level higher than Next.js. A Blitz codebase is not a *lot* different than a Next.js codebase. But it is enough different that it wouldn't be super simple to "eject" to plain Next.js. For example, you'll probably use the Blitz authentication which is unique to Blitz. But switching to Next Auth would give you a comparable experience with Next.js.
+De mate van lock-in bij Blitz is groter dan bij Next.js. Een Blitz-codebase verschilt niet zoveel van een Next.js-codebase, maar het verschil is groot genoeg dat het niet super eenvoudig is om een Blitz applicatie om te zetten naar een gewone Next.js applicatie. Je zal waarschijnlijk gebruik maken van de authenticatie die eigen is aan Blitz. Overschakelen naar NextAuth zou je een vergelijkbare ervaring geven met Next.js.
 
-The Blitz zero-API data layer is unique to Blitz. So if switching to Next.js, you'd have to manage your own API code (which you would have to do if you started with Next.js, so you haven't lost anything). However, all your backend query and mutation logic can be directly ported.
+De Zero-API-datalaag is uniek voor Blitz. Dus als je overstapt op Next.js, moet je jouw eigen API-code beheren (wat je ook zou moeten doen als je met Next.js zou beginnen, dus je hebt niets verloren). Al je backend query- en mutatielogica kan echter rechtstreeks worden overgezet.
 
+## Next.js upgrades {#next-js-upgrades}
 
-## Next.js Upgrades {#next-js-upgrades}
+Als je op het scherpst van de snede wilt zitten wat Next.js-releases betreft, weet dan dat Blitz iets achter zal lopen, maar niet ver. Gewoonlijk updaten we Blitz binnen een week of minder na een nieuwe Next.js release.
```

Comment:
Ik weet niet of _op het scherpst van de snede_ hier de beste uitdrukking voor is. Ik zou eerder iets zeggen als: _Als je steeds met de nieuwste versie van Next.js wil werken, weet dan ..._